Adyen/UI/List/ListItemView.swift

```diff
@@ -10,11 +10,17 @@ import UIKit
 /// Displays a list item.
 @_spi(AdyenInternal)
 public final class ListItemView: UIView, AnyFormItemView {
```

Comment:
Do we have control over it's lifecycle? Ideally we want to cancel image loading when the view will disappear.

Reply:
We could use `didMoveToWindow()` to check whether it's visible 🤔

Reply:
Then let's go to the level up? This is UIView so it should be managed by either a view controller or table/collection view which have specific methods to let us propagate willShow/willDisappear events.
This goes quite a bit beyond the initial scope though.

Reply:
I agree - but to keep the scope small I'd think of how to best do it in a subsequent PR